CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## 4.6
 
+- [#928](https://github.com/openshift/cluster-monitoring-operator/pull/928) Bump prometheus-operator 0.42:
+  - 0.42.0 changes: https://github.com/prometheus-operator/prometheus-operator/releases/tag/v0.42.0
+  - 0.41.1 changes: https://github.com/prometheus-operator/prometheus-operator/releases/tag/v0.41.1
+  - 0.41.0 changes: https://github.com/prometheus-operator/prometheus-operator/releases/tag/v0.41.0
```

Comment:
is this sufficient for us? :thinking:

Reply:
Yes even just bump to Prometheus operator 0.42 would be enough. 👍